### PR TITLE
feat(EU): support new hazard lights and horn features

### PR DESCRIPTION
**Pull request by emtunc**
https://github.com/Hyundai-Kia-Connect/kia_uvo/issues/957

Attempting to add new "Hazards" and "Hazards and Horn" feature. My first contribution so will likely need some help getting it release-worthy.

@cdnninja @asychow 🙏

### CODE REVIEW

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 1]
```diff
@@ -1035,6 +1035,40 @@ def stop_charge(self, token: Token, vehicle: Vehicle) -> str:
...
+        url = self.SPA_API_URL_V2 + "vehicles/" + vehicle.id + "/ccs2/control/hornlight"
+
+        payload = {"command": "on"}
+        _LOGGER.debug(f"{DOMAIN} - Start Hazard Lights and Horn Request: {payload}")
+        response = requests.post(
+            url,
+            json=payload,
+            headers=self._get_authenticated_headers(
+                token, vehicle.ccu_ccs2_protocol_support
+            ),
```
Comment (asychow) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:1065]: I am getting `Unknown API response format: {'error': 'Access to this API has been disallowed'}`

but it works fine after change the 'headers' from self._get_authenticated_headers to 

```
            headers=self._get_authenticated_headers(
                token, vehicle.ccu_ccs2_protocol_support
            ),
```

to 

```
headers=self._get_control_headers(
                token, vehicle
            ),
```

It should then replicate the PUT request for the pin before the hazard light command.

Comment (emtunc) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: Updated - can you check now for me @asychow? I'm still rate limited.

Comment (asychow) [reply hyundai_kia_connect_api/KiaUvoApiEU.py]: Works on my end :)

**Comment by cdnninja** [comment]
Looks great!   Have you been able to test it? I'll merge either way.  Next step is calling this via services.

**Pull request merged**